draft-ietf-quic-transport.md

```diff
@@ -124,10 +124,54 @@ provides:
 
 * Authenticated and encrypted header and payload
 
-QUIC uses UDP as a substrate to avoid requiring changes to legacy client
-operating systems and middleboxes.  QUIC authenticates all of its headers and
-encrypts most of the data it exchanges, including its signaling, to avoid
-incurring a dependency on middleboxes.
+QUIC establishes a connection, which is a stateful interaction between a client
+and server. The primary purpose of a connection is to support the exchange of
+data by an application protocol.
+
+Streams ({{streams}}) are the primary means by which an application protocol
+exchanges information.  Streams are ordered sequences of bytes.  Two types of
+stream can be created: bidirectional streams allow both endpoints to send data;
+unidirectional streams allow a single endpoint to send.  Endpoints can create
+streams without prior negotiation.  A credit-based scheme is used to limit
```

Comment:
I'd drop this sentence, since one could consider the stream limit TPs 'negotiation'.

```suggestion
unidirectional streams allow a single endpoint to send.  A credit-based scheme is used to limit
```